jujugui/static/gui/src/app/jujulib/bakery-factory.js

```diff
@@ -0,0 +1,60 @@
+/* Copyright (C) 2016 Canonical Ltd. */
+'use strict';
+
+var module = module;
+
+(function (exports) {
+
+
+  var jujulib = exports.jujulib;
+  /**
+   * BakeryFactory to create and maintain bakery objects.
+   *
+   */
+
+  /**
+   Initialize.
+
+   @method initializer
+   @param {Object} cfg a config object containing the Bakery module
```

Comment:
If it's only for a field, couldn't we just pass that field? For instance `function bakeryFactory(bakeryModule)` ?